gers/registry/registry_schema.yaml

```diff
@@ -0,0 +1,53 @@
+"$schema": https://json-schema.org/draft/2020-12/schema
+title: Global Entity Reference System Registry Schema
+description: >-
+  GERS Registry Schema
+type: object
+unevaluatedProperties: false
+required:
+  - id
+  - first_seen
+  - last_seen
+properties:
+  id:
+    "$ref": ../../schema/defs.yaml#/$defs/propertyDefinitions/id
+  bbox:
+    title: Bounding Box
+    desdcription: |
```

Comment:
is "the envelope of the entity's spatial extent" the description we should be using for all instances of the `bbox` property?